src/actions/actions/services/dispatchers/verusid/verusid.js

```diff
@@ -191,10 +192,18 @@ export const checkVerusIdNotificationsForUpdates = async () => {
         let errorFound = false;
         try {
           if (pendingIds[ticker][iaddress].infoUri) {
+            
             const response = await axios.get(pendingIds[ticker][iaddress].infoUri);
-            const responseData = new primitives.LoginConsentProvisioningDecision(response.data);
+            const responseData = new primitives.LoginConsentProvisioningResponse(response.data);
+            const req = new primitives.LoginConsentRequest();
+            req.fromBuffer(Buffer.from(pendingIds[ticker][iaddress].loginRequest, 'base64'));
+            const verified = await verifyIdProvisioningResponse(system, response.data);
+
+            if (responseData.signing_id !== req.signing_id && !verified) {
```

Comment:
Should be an "or" statement, otherwise it will take both cases of failing verification to actually fail.